README.md

```diff
@@ -28,6 +28,10 @@ $ make
 Command line usage requires that you have the necessary software installed.  See
 [the instructions](https://github.com/martinthomson/i-d-template/blob/main/doc/SETUP.md).
 
+# Implementations
+
+- TypeScript: [OWF sd-jwt-js](https://github.com/openwallet-foundation/sd-jwt-js/tree/main/packages/jwt-status-list)
```

Comment:
In my point of view, the repo is would be better: https://github.com/openwallet-foundation/sd-jwt-js
What do you guys think?

Reply:
sure I don't mind